ui/js/perf.js

```diff
@@ -4,14 +4,20 @@ import uiBootstrap from 'angular1-ui-bootstrap4';
 import uiRouter from '@uirouter/angularjs';
 import 'ng-text-truncate-2';
 import LocalStorageModule from 'angular-local-storage';
+import { react2angular } from 'react2angular/index.es2015';
 
+import Login from '../shared/Login';
```

Comment:
Sorry for not spotting this previously - but now Login is being imported in this file, the import in `entry-perf.js` can be removed:
https://github.com/mozilla/treeherder/blob/7c8a0b5e6e50435c385f6a8ea658f4354af30efd/ui/entry-perf.js#L25-L26